guide/src/module.md

```diff
@@ -1,44 +1,49 @@
 # Python Modules
 
-You can create a module as follows:
+You can create a module using `#[pymodule]`:
```

Comment:
This is something we picked up on with docs for Pants: for bigger code blocks, preview what the main thing to look for is.